docs/source/pipelines.rst

```diff
@@ -118,38 +119,106 @@ Key concepts
-   README file for information about the :ref:`Pipeline Steps <steps-key-concept>` that it defines
-   and the results that it produces.
+   problem that you wish to solve. Follow the template's README file for template-specific
+   instructions.
+
+    - [Local] See example below for cloning the |MLflow Pipelines Regression Template| on a local
+      machine.
 
     .. code-block:: sh
-      :caption: An example of cloning the |MLflow Pipelines Regression Template|
 
       git clone https://github.com/mlflow/mlp-regression-template
 
-    .. note::
-      On Databricks, we recommend cloning the :ref:`Pipeline Template <pipeline-templates>` git
-      repository using |Databricks Repos|.
+    - [Databricks] Clone the |MLflow Pipelines Regression Template| git repository using |Databricks Repos|.
 
       .. image:: _static/images/pipelines_databricks_repo_ui.png
         :width: 60%
 
-2. Run the pipeline and inspect its results. When a pipeline run completes, MLflow Pipelines
+2. Edit required fields by searching comments ``FIXME::REQUIRED`` in ``pipeline.yaml`` and
+   ``profiles/*.yaml``. The pipeline is runnable once all required fields are filled with
+   proper values. You may proceed to step 3 if this is the first time going through this step.
+   Otherwise, continue to edit the YAML config files as well as ``steps/*.py`` files,
+   filling out areas marked by ``FIXME::OPTIONAL`` as you see fit to
+   customize the pipeline steps to your ML problem for better model performance.
+
+      .. image:: _static/images/pipelines_databricks_fixme.png
+        :width: 60%
+
+3. Run the pipeline and inspect its results. When a pipeline run completes, MLflow Pipelines
    creates and displays an interactive **Step Card** with the results of the last executed
    :ref:`step <steps-key-concept>`.
+   Each :ref:`Pipeline Template <pipeline-templates>` also includes a |Databricks Notebook|
+   and a |Jupyter Notebook| for running the pipeline and inspecting its results.
 
     .. figure:: _static/images/pipelines_evaluate_step_card.png
-      :scale: 25
+      :width: 60%
 
       An example step card produced by running the **evaluate** step of the
       :ref:`MLflow Regression Pipeline <mlflow-regression-pipeline>`. The step card results
       indicate that the trained model passed all performance validations and is ready for
       registration with the :ref:`MLflow Model Registry <registry>`.
 
+
+    .. figure:: _static/images/pipelines_databricks_logged_artifacts.png
+      :width: 60%
+
+      An example MLflow run view page, showing that artifacts from pipeline steps are mostly logged
+      and sharable through the MLflow experiment.
+ 
+    .. note::
+      Data profiling is often best viewed with "quantiles" mode. To switch it on, on the Facet
+      data profile, find ``Chart to show``, click the selector below, and choose ``Quantiles``.
+
     |
 
     .. code-section::
```

Comment:
Can we move the code section to the top of step 3 so that the results are shown after the MLP execution code? I think most users are probably looking for info about what code to run first.

Reply:
Done. Also moved NOTE to the last part of the step 3.